docs/sources/configure-client/grafana-agent/ebpf/configuration.md

```diff
@@ -0,0 +1,134 @@
+
+The `forward_to` parameter should point to a `pyroscope.write` component to send the collected profiles to.
+
+| Name                      | Type                     | Description                                                  | Default | Required |
+|---------------------------|--------------------------|--------------------------------------------------------------|---------|----------|
+| `targets`                 | `list(map(string))`      | List of targets to group profiles by container id            |         | yes      |
+| `forward_to`              | `list(ProfilesReceiver)` | List of receivers to send collected profiles to.             |         | yes      |
+| `collect_interval`        | `duration`               | How frequently to collect profiles                           | `15s`   | no       |
+| `sample_rate`             | `int`                    | How many times per second to collect profile samples         | 97      | no       |
+| `pid_cache_size`          | `int`                    | The size of the pid -> proc symbols table LRU cache          | 32      | no       |
+| `build_id_cache_size`     | `int`                    | The size of the elf file build id -> symbols table LRU cache | 64      | no       |
+| `same_file_cache_size`    | `int`                    | The size of the elf file -> symbols table LRU cache          | 8       | no       |
+| `container_id_cache_size` | `int`                    | The size of the pid -> container ID table LRU cache          | 1024    | no       |
+| `collect_user_profile`    | `bool`                   | A flag to enable/disable collection of userspace profiles    | true    | no       |
+| `collect_kernel_profile`  | `bool`                   | A flag to enable/disable collection of kernelspace profiles  | true    | no       |
+| `demangle`                | `string`                 | C++ demangle mode. Available options are: `none`, `simplified`, `templates`, `full` | `none` | no |
+
+## Supported languages
+
+The `pyroscope.ebpf` component supports the following languages:
+
+- Go
+- Rust
+- C/C++ with frame pointers enabled
+- Python
+
+## Sending data Grafana Cloud Profiles
+
+When sending to Grafana Cloud Profiles, you can use the following `pyroscope.write` component configuration which makes uses of environment variables:
+
+```river
+pyroscope.write "endpoint" {
+    endpoint {
+        basic_auth {
+            password = env("GC_PASSWORD")
+            username = env("GC_USER")
+        }
+        url = env("GC_URL")
+    }
+}
+```
+
+Ensure that you have appropriately configured the `GC_URL`, `GC_USER`, and `GC_PASSWORD` environment variables.
+
+## Exported fields
+
+`pyroscope.ebpf` does not export any fields that can be referenced by other
+components.
+
+## Component health
+
+`pyroscope.ebpf` is only reported as unhealthy if given an invalid
+configuration.
+
+## Debug information
```

Comment:
maybe a dumb question, but are these configuration options or something that you get associated with each profile? or something else? probably more of an explanation of when someone would see these / what they would use them for